src/web/routes/application/children-dob/selectors.js

```diff
@@ -0,0 +1,14 @@
+const { pickBy } = require('ramda')
+const { isNotChildEntry, isChildEntry, keyDoesNotContainIndex } = require('./predicates')
+
+const getNonChildEntries = pickBy(isNotChildEntry)
+
+const getChildEntries = pickBy(isChildEntry)
+
+const getEntriesWithoutIndex = (entries, index) => pickBy(keyDoesNotContainIndex(index), entries)
```

Comment:
This could be `omitKeysWithIndex` as this is similar to the function of Ramda's `omit` .